### PR TITLE
Added the select a package page

### PR DESCRIPTION
**Pull request by Peartes**
[x] Added UI components

### CODE REVIEW

File: src/app.ts [hunk 1]
```diff
@@ -1,3 +1,4 @@
+/* eslint-disable linebreak-style */
```
Comment (dkent600) [inline src/app.ts:1]: why is this added?

File: src/selectPackage/selectPackage.ts [hunk 2]
```diff
@@ -1,3 +1,26 @@
+/* eslint-disable linebreak-style */
 import "./selectPackage.scss";
+
 export class SelectPackage {
+  // Declare state variables
+  benefits: Array<string>;
+  discounts: { [plan:string]:{discount: number, benefits: Array<string>}};
```
Comment (dkent600) [inline src/selectPackage/selectPackage.ts:7]: Why this complication in the ts?  Can this just be hardcoded in the HTML?  Same with benefits.

File: src/selectPackage/selectPackage.scss [hunk 3]
```diff
@@ -1,3 +1,97 @@
...
+    //   font-size: 0.7em;
+    //   border-radius: 20px;
+    //   // TODO: Colors need to be updated in global style
+    //   &:hover {
+    //     cursor: pointer;
+    //   }
+    // }
+    a {
+      font-size: 0.8em;
+      text-decoration: underline;
```
Comment (dkent600) [inline src/selectPackage/selectPackage.scss:76]: underline should be the default for all anchor.

File: src/selectPackage/selectPackage.scss [hunk 4]
```diff
@@ -1,3 +1,97 @@
...
+      // TODO: Wrong color
+      text-decoration-color: #2e5aff;
+      text-decoration-thickness: 0.15em;
```
Comment (dkent600) [inline src/selectPackage/selectPackage.scss:79]: hmmm, is this really needed?  it is very unusual.

File: src/selectPackage/selectPackage.scss [hunk 5]
```diff
@@ -1,3 +1,97 @@
...
+          color: firebrick;
+          font-size: 1.3em;
+          padding: 0.4em;
+        }
+      }
+    }
+    .buttonContainer a {
+      text-decoration: none;
+      // TODO: Remove this
+      color: white;
```
Comment (dkent600) [inline src/selectPackage/selectPackage.scss:59]: I think we should always use color variables from _colors.scss.

File: src/selectPackage/selectPackage.scss [hunk 6]
```diff
@@ -1,3 +1,97 @@
 @import "styles/colors.scss";
+@import "styles/styles";
```
Comment (dkent600) [inline src/selectPackage/selectPackage.scss:2]: this should not need to be imported

**Review by dkent600 — CHANGES_REQUESTED** [review]

File: src/selectPackage/selectPackage.ts (hunk 2 above)
Comment (dkent600) [reply src/selectPackage/selectPackage.ts:7]: aside from the complication of add'l code, realize that aurelia will create observables on these unless you specify them to be "one-time".  but then again, it begs the question why have them at all?

File: src/app.ts (hunk 1 above)
Comment (Peartes) [reply src/app.ts:1]: 🤦🏽‍♂️  it's because of difference in our linebreak char code. My linter always marks the line-break as an error. That is just to make it stop

Comment (dkent600) [reply src/app.ts:1]: go ahead and delete this line and pay no attention to the lint error for now. I will convert everything to LF and the error will go away for you.

**Review by dkent600 — CHANGES_REQUESTED** [review]

File: src/app.ts (hunk 1 above)
Comment (Peartes) [reply src/app.ts:1]: 👌🏽

File: src/selectPackage/packageContainer.html [hunk 7]
```diff
@@ -0,0 +1,29 @@
...
+        <div class="benefit"><span>&#10026;</span> Hosted</div>
+        <div class="benefit"><span>&#10026;</span> Marketing</div>
+        <div class="benefit">
+          <span>&#10026;</span> 25% discount on services
+        </div>
+      </div>
+    </div>
+    <div class="packageFooter">
+      <div class="heading heading2 discount">
+        ${discount > 0 ? discount + "% "+ "FEE" : "FREE"}
```
[the remark this answers is not in the document]
Comment (Peartes) [reply src/selectPackage/packageContainer.html:17]: 👌🏽

**Review by dkent600 — APPROVED** [review]
